src/main/scala/com/actian/spark_vector/vector/VectorJDBC.scala

```diff
@@ -78,6 +115,13 @@ class VectorJDBC(cxnProps: VectorConnectionProperties) extends Logging {
   def executeQuery[T](sql: String)(op: ResultSet => T): T =
     withStatement(statement => managed(statement.executeQuery(sql)).map(op)).resolve()
```

Comment:
[not related to this CR]
can't this scala-arm managed lib replace our custom ResourceUtil stuff?